mesh/profiles/urls.py

```diff
@@ -1,9 +1,12 @@
-from django.urls import path
+from django.urls import path, include
 from . import views as profile_views
 from django.http import HttpRequest
+from .views import *
```

Comment:
The `from .views import *` seems like it would be redundant when `from . import views as profile_views`

Reply:
I generally dont suggest importing all. I suggest importing only what's necessary.